proxy/proxy.go

```diff
@@ -1335,6 +1335,7 @@ func (p *Proxy) do(ctx *context, parentSpan ot.Span) (err error) {
 		}
 
 		ctx.setResponse(rsp, p.flags.PreserveOriginal())
+		ctx.backendTime = time.Since(backendStart)
```

Comment:
Lets discuss. It feels like this is collected by two lines above and also it does not have request host dimension (ctx.metricsHost())